crates/fluvio-spu/src/replication/leader/replica_state.rs

```diff
@@ -46,6 +50,7 @@ pub struct LeaderReplicaState<S> {
     followers: Arc<RwLock<BTreeMap<SpuId, OffsetInfo>>>,
     status_update: SharedStatusUpdate,
     sm_ctx: Option<SharedSmartModuleContext>,
+    consumer_offset_publishers: Arc<Mutex<Vec<Arc<OffsetPublisher>>>>,
```

Comment:
alias `Arc<OffsetPublisher>` so it's easier to to read

Reply:
Added `SharedOffsetPublisher` and `WeakSharedOffsetPublisher` aliases